src/types/daisyui.type.ts

```diff
@@ -0,0 +1,27 @@
+import {
```

Comment:
Potentially we should also delete this when we install `react-daisyui`, since it would be imported there.